src/sentry/tasks/integrations/github/pr_comment.py

```diff
@@ -100,3 +104,45 @@ def get_comment_contents(issue_list: List[int]) -> List[PullRequestIssue]:
         PullRequestIssue(title=issue.title, subtitle=issue.message, url=issue.get_absolute_url())
         for issue in issues
     ]
+
+
+def comment_workflow():
+    pr_list = pr_to_issue_query()
+    for (gh_repo_id, pr_key, org_id, issue_list) in pr_list:
+        try:
+            organization = Organization.objects.get_from_cache(id=org_id)
+        except Organization.DoesNotExist:
```

Comment:
Why are you silently overlook these exception? If this is unexpected to happen I suggest log this somewhere so we get notified when something goes wrong.

Reply:
we use soft foreign keys extensively. It's not uncommon to have dangling integrations. we just don't want to crash on them.

Reply:
Logging is a good idea though.

Reply:
i could log these to GCP but i don't think it should be erroring because this task may be called in postprocess for all PRs and issues in the worst case